metadata-ingestion/tests/integration/powerbi/test_m_parser.py

```diff
@@ -240,7 +249,7 @@ def test_databricks_regular_case():
         ctx=ctx,
         config=config,
         platform_instance_resolver=platform_instance_resolver,
-    )
+    )[0].upstreams
```

Comment:
shouldn't we also check the CLL for some of these test case?

Reply:
updated `test_sqlglot_parser` test-case